tox.ini

```diff
@@ -7,7 +7,9 @@ envlist =
 setenv =
     PYTHONDONTWRITEBYTECODE=1
 install_command = pip install --no-deps {opts} {packages}
-deps = -rrequirements.txt
+deps = 
+    -rrequirements.txt
+    py27: ipaddress
```

Comment:
ipaddress is a dependency of faker. Anyway, I recompiled requirements.txt using python2.7, and removed this one from tox.ini.